### ISSUE TITLE
Product list not appearing in frontend

### ISSUE THREAD

**Issue by amruthalakshmiarun** [pixelant/pxa_product_manager#398]
I have products in my backend. but it is not displayed on the frontend. This happened when I update typo3 10 to 11. I have chosen the product manager lazy list in the backend.

**Comment by MattiasNilsson** [comment]
@amruthalakshmiarun Do you have any more information regarding this? Error logs?

**Comment by amruthalakshmiarun** [comment]
@MattiasNilsson, the extension is working on typo3 version 10. I have updated typo3 to 11 and also updated the pxa_product_manager to the latest version. I am trying to filter the products by attributes. When i add the product in backend, there is no options to choose the attributes. Is that the problem? 
https://www.awesomescreenshot.com/image/37331364?key=4e792492c43ccfcdcbfa66dee4a8981b

**Comment by amruthalakshmiarun** [comment]
@MattiasNilsson , This is the error log.
https://tinyurl.com/2fvot2q3

**Comment by amruthalakshmiarun** [comment]
@MattiasNilsson, There is no errors in error log.

**Comment by MattiasNilsson** [comment]
@amruthalakshmiarun Have you tried to clear the cache for the whole system? And then save the Product afterwards?

**Comment by amruthalakshmiarun** [comment]
Yes @MattiasNilsson, I have cleared all the cache. I can see all attributes in the list but not in the attribute tab on each product edit. [attribute values] still seen empty.
